src/Meshes/thinInstanceMesh.ts

```diff
@@ -305,7 +305,7 @@ Mesh.prototype.thinInstanceRefreshBoundingInfo = function (forceRefreshParentInf
 
     if (forceRefreshParentInfo) {
         vectors.length = 0;
-        this.refreshBoundingInfo(true);
+        this.refreshBoundingInfo();

```

Comment:
So, if the mesh has a skeleton, we don't take it into account anymore to compute the bounding box? Is it for a performance reason? Maybe we can pass a flag to `thinInstanceRefreshBoudingInfo` that would make the call to `refreshBoundingInfo` pass `true, true` as parameters?

Reply:
I agree we should have the same set of parameters so the user can decide if they want to take skeleton and morph in account